libexec/cli/pull.exec

```diff
@@ -151,8 +151,13 @@ case "$SINGULARITY_CONTAINER" in
             fi
         fi
 
-         if [ -x "${SINGULARITY_bindir}/singularity" ]; then
-             ${SINGULARITY_bindir}/singularity build ${SINGULARITY_IMAGE} ${SINGULARITY_CONTAINER}
+        DBGFLAG=""
+        if [ "x${SINGULARITY_MESSAGELEVEL}" == "x5" ]; then
```

Comment:
Why does this say `"x$...` and `"x5"` instead of `"$...` and `"x5"`?

Reply:
Because if `SINGULARITY_MESSAGELEVEL` is not set, you _could_ end up like:
```
if [ == 5 ]; then
```
And not have a left hand side. So ... it's common practice to add an `x` in, so the check ends up as:
```
if [ x == x5 ]; then
```